Headers/DebugServer2/Types.h

```diff
@@ -146,6 +146,7 @@ struct StopInfo {
     kReasonThreadSpawn,
     kReasonThreadEntry,
     kReasonThreadExit,
+    kReasonClone,
```

Comment:
I think you're duplicating what `kReasonThreadSpawn` should be. We shouldn't be adding this. You'll also see that `kReasonThreadSpawn` exists but isn't used anywhere.

As a side note, if you add something here, you should be adding it to `Stringify::StopReason` as well. You won't have to care about this if you directly use `kReasonThreadSpawn` though.

Reply:
👍